ovis_bringup/launch/ovis_moveit.launch

```diff
@@ -0,0 +1,31 @@
+<launch>
+    <!-- Declare args to be passed in -->
+    <arg name="rviz" default="true"/> 
+    <arg name="ovis" default="true"/> 
+    <arg name="moveit" default="true"/> 
+    <arg name="jog" default="true"/> 
+    <arg name="action" default="true"/> 
```

Comment:
For debug (multiple terminal) and because we used to only have one launch file for sim/real.

Examples: 
- You don't want jog, if you don't have a controller (you only want to move with rviz)
- You don't want action if you use gazebo (action server conflict)
- You might want to launch ovis_arm in an other terminal so it stay active when testing

Reply:
Might want to consider creating specific launch files or at least commenting this information in the file.